### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by GurgenBaghdasaryan**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import Footer from "./footer";
+import "./App.css";
+
+function App() {
+  const [number, setNumbers] = useState([]);
+
+  const deleteFromNumer = (index) => {
+    const newArr = [...number];
+    newArr.splice(index, 1);
+    setNumbers(newArr);
```
Comment (edotnet) [inline src/App.js:13]: create new function where new array will be set and localstorage updated to not set localstorage item in every function. Use the function everywhere

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+  const Clear = () => {
+    const newArr = [...number];
+    newArr.splice(0);
+    setNumbers(newArr);
+    localStorage.setItem("num", JSON.stringify(newArr));
+  };
+
+  const addCard = () => {
+    const newArr = [];
```
Comment (edotnet) [inline src/App.js:25]: use [...number]

File: src/App.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    const row = JSON.parse(localStorage.getItem("num"));
+    setNumbers(row);
+  }, []);
+
+  return (
+    <>
+      <div>
+        <Header addCard={addCard} sortCards={sortCards} Clear={Clear} />
+        <div className="container content">
+          {number.map((num, index) => (
```
Comment (edotnet) [inline src/App.js:49]: check if number is defined `number && number.map`

File: src/App.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+  const sortCards = () => {
+    const newArr = [...number];
+    newArr.sort((a, b) => b - a);
+    setNumbers(newArr);
+    localStorage.setItem("num", JSON.stringify(newArr));
+  };
+
+  useEffect(() => {
+    const row = JSON.parse(localStorage.getItem("num"));
```
Comment (Shitikyan) [inline src/App.js:40]: check is the element in local storage before putting it on json parse

File: src/Card/style.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  height: 161px;
+  width: 65%;
+  list-style: none;
+  border: 1px solid silver;
+  text-align: center;
+`;
+
+export const StyledButton = styled.button`
+  margin-top: 7px;
+  margin-left: 92%;
```
Comment (Shitikyan) [inline src/Card/style.js:13]: Please do not use magic numbers for the css

File: src/Card/index.js [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+
+const Card = ({ number, index, deleteFromNumer }) => {
+  return (
+    <>
+      <ul key={index}>
+        <StyledCard>
+          <StyledButton type="checkbox" onClick={() => deleteFromNumer(index)}>
+            x
+          </StyledButton>
+          <div style={{ marginTop: "45px" }}>{number}</div>
```
Comment (Shitikyan) [inline src/Card/index.js:12]: Remove the style tag

**Review by Shitikyan — CHANGES_REQUESTED** [review]

**Pull request merged**